src/sfctl/custom_chaos.py

```diff
@@ -6,6 +6,62 @@
 
 """Custom commands for the Service Fabric chaos service"""
 
+def parse_chaos_parameters(chaos_parameters):
+    #pylint: disable=too-many-locals
```

Comment:
Does this work if you put the disable at the end of the line above? Or does it make the line too long?